lib/src/models/album.dart

```diff
@@ -105,3 +109,5 @@ class AlbumSimple extends Object {
 }
 
 enum DatePrecision { day, month, year }
+
+enum AlbumType { album, singe, compilation }

```

Comment:
Let's make it a `single` 😄 

```suggestion
enum AlbumType { album, single, compilation }
```